bedrock/privacy/templates/privacy/index.html

```diff
@@ -48,9 +48,13 @@ <h2>{{ _('Contact Mozilla') }}</h2>
           <span itemprop="streetAddress">331 E. Evelyn Ave</span>,<br>
           <span itemprop="addressLocality">Mountain View</span>,
           <span itemprop="addressRegion">CA</span>
-          <span itemprop="postalCode">94041</span>
+          <span itemprop="postalCode">94041</span><br>
         </span>
       </p>
+      <a href="mailto:compliance@mozilla.com">compliance@mozilla.com</a>
+      {% if l10n_has_tag('privacy_email_2019') %}
+      <p>See <a href="https://app.onetrust.com/app/#/webform/4ba08202-2ede-4934-a89e-f0b0870f95f0">here</a> for Data Subject Access Requests.</p>
```

Comment:
This string needs to be wrapped for l10n, either in a `{% trans %}` block or `_(' ')`

Reply:
whoops! Thank you!